config.md

```diff
@@ -346,3 +346,28 @@ Hover:
 
 A boolean that controls printing of desugared types, e.g:
 `vector<int>::value_type (aka int)`
+
+## Semantic Tokens
+{:.v17}
+
+Configure semantic highlighting. Sample block (default):
+
+```yaml
+SemanticTokens:
+  DisabledKinds: []
+  DisabledModifiers: []
+```
+
+### DisabledKinds
+{:.v17}
+
+Specify semantic token kinds that clangd should not send to client.
+
+Available kinds could be found [here](features). Note one exception is `InactiveCode`, which doesn't work with this config.
```

Comment:
(By the way, this is an aside, but I've been using your [minimal C++ syntax](https://marketplace.visualstudio.com/items?itemName=daiyousei-qz.minimal-cpp-syntax) extension, and I realized that it has a downside: it means that even with https://github.com/clangd/vscode-clangd/pull/193, inactive code will not be any more colorful than before.

I wonder if it's possible to have a kind of extension that behaves like [better C++ syntax](https://marketplace.visualstudio.com/items?itemName=jeff-hykin.better-cpp-syntax) for inactive code (heuristically assigning client-side colors to the tokens), but behaves like "minimal C++ syntax" for active code (since the language server takes care of that.)

Reply:
> You can link more specifically to the relevant section using an HTML anchor: `[here](features#kinds)`

Was wondering how to do such jump. Added anchor.